src/faldbt/lib.py

```diff
@@ -558,6 +561,32 @@ def _existing_or_new_connection(
 
 
 # Adapter: BigQuery
+def _bigquery_execute_sql(
+    adapter: BaseAdapter, sql: str, open_conn: bool
+) -> Tuple[AdapterResponse, pd.DataFrame]:
+    assert adapter.type() == "bigquery"
+
+    import google.cloud.bigquery as bigquery
+
+    # HACK: we need to include uniqueness (UUID4) to avoid clashes
+    name = "bigquery:execute_sql:" + str(hash(sql)) + ":" + str(uuid4())
+    with _existing_or_new_connection(adapter, name, open_conn):
```

Comment:
This doesn't cause conflicts if two execute_sql calls are running simultaneously? Or if a write is running?